documentation/usersguide/whatsnew.md

```diff
@@ -12,6 +12,11 @@ title: What's New
 #### Upcoming next release
 See upcoming changes from the next version on [github](https://github.com/naemon/naemon-core/blob/master/NEWS)
 
+#### 1.2.1 - 13 Jul 2020
+##### Bugfixes
+  * fix executing commands with pipes
+  * fix external command to remove downtimes by filter
+
```

Comment:
Should we have a section for Livestatus here?

Reply:
you are right, added the missing section